spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/ops/DecodeImage.scala

```diff
@@ -145,7 +145,7 @@ class DecodeGif[T: ClassTag]()(implicit ev: TensorNumeric[T])
 }
 
 class DecodeRaw[T: ClassTag](val outType: DataType,
-                             val byteOrder: ByteOrder)(implicit ev: TensorNumeric[T])
+                             val littleEndian: Boolean)(implicit ev: TensorNumeric[T])
```

Comment:
please add a comment to explain why change it to boolean

Reply:
ByteOrder is not serilizable